### PR TITLE
Presentation: DevOps practises in a Sundsvall company

### PR DESCRIPTION
**Pull request by heeenkie**
For weak 2.
more information in the readme

### CODE REVIEW

File: contributions/presentation/weak2/isacarv-anhenri/README.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+ Andreas Henriksson (anhenri@kth.se)
+ Github username: [heeenkie](https://github.com/heeenkie)
+
+ ## Proposal
+ Docker vs Kubernetes
+
+-introduction to containerization.
+-A brief summary of the two tools.
+-Differens between the tools
```
Comment (cesarsotovalero) [inline contributions/presentation/weak2/isacarv-anhenri/README.md]: It would be better to focus on one specific aspect of one of the tools.

**Review by cesarsotovalero — CHANGES_REQUESTED** [review]

File: contributions/presentation/weak2/isacarv-anhenri/README.md (hunk 1 above)
Comment (heeenkie) [reply contributions/presentation/weak2/isacarv-anhenri/README.md]: So would just a presentation of the docker tool do it?

-information about containerization
-what is the docker tool
-usecases

Comment (cesarsotovalero) [reply contributions/presentation/weak2/isacarv-anhenri/README.md]: @heeenkie a presentation about the basics of Docker is not enough. 
Docker is not new anymore. You need to focus your presentation on a specific Docker user case in detail.

**Review by heeenkie — COMMENTED** [review]
@cesarsotovalero A new proposal of the presentation

**Review by cesarsotovalero — APPROVED** [review]

**Comment by cesarsotovalero** [comment]
Hi @heeenkie, your new proposal LGTM. 
I'll merge.

**Pull request merged**